tests/packages/mlx/test_mellanox-modules-compat.py

```diff
@@ -0,0 +1,19 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): any master host of a pool, with access to XCP-ng RPM repositories and reports.xcp-ng.org.
+import logging
+from pkgfixtures import host_with_saved_yum_state
+
+def test_install_mlx_modules_alt(host_with_saved_yum_state):
+    host = host_with_saved_yum_state
+    logging.info(f"Installing mlx4-modules-alt alone")
+    host.yum_install(['mlx4-modules-alt'])
+    host.ssh(['modprobe', 'mlx4_ib'])
+    host.ssh(['rmmod', 'mlx4_ib'])
```

Comment:
Why not use `modprobe -r` here?

Also, we should remove from memory modules that were pulled by mlx4_en, especially `mlx_compat.ko` which is the one that might become incompatible someday. Otherwise they will remain and cause the next execution of these tests to be inaccurate.